### DISCUSSION TITLE
How can I use regular expressions?

### DISCUSSION THREAD

**Discussion by acentrist** [filamentphp/filament#3448] [Help] [answered]
What I want is a simple regular expression,
it like simple regular expression that checks if a 10-digit number has been entered.
Additionally, I want to use it as a validation in my create and edit forms.

**Comment by danharrin** [comment, marked as answer]
I think we have a regex() method for TextInput which you could use for the validation?

**Reply by acentrist** [reply]
> I think we have a regex() method for TextInput which you could use for the validation?

Thanks...
i was completely blind...
